modules/monitoring-about-configuring-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc
+
+:_content-type: CONCEPT
```

Comment:
```suggestion
:_content-type: REFERENCE
```
The content makes this seem like more of a reference module : https://redhat-documentation.github.io/modular-docs/#creating-reference-modules